src/main/java/com/uber/cadence/internal/replay/SignalExternalWorkflowParameters.java

```diff
@@ -94,10 +94,20 @@ public SignalExternalWorkflowParameters withWorkflowId(String workflowId) {
 
   @Override
   public String toString() {
+
```

Comment:
this whole thing seems utterly pointless and not good, but fixing it up since god forbid, someone mgiht actually be caring about it's current not good structure. 

the main issue is that before, it broke for short strings, which is probably not what you want for a tostring